.circleci/config.yml

```diff
@@ -422,6 +422,32 @@ jobs:
           background: true
           paths:
             - ./habitat-api
+      - run:
+          name: Build api documentation
```

Comment:
We also need to add this (minus the cache saving) to habitat-api to make sure PRs there don't break the doc built for habitat-api, which would then break habitat-sim's CI.

Reply:
https://github.com/facebookresearch/habitat-api/pull/393